repo/caching.go

```diff
@@ -75,6 +76,11 @@ func setupCachingOptionsWithDefaults(ctx context.Context, configPath string, lc
 	lc.Caching.MinContentSweepAge = opt.MinContentSweepAge
 	lc.Caching.MinMetadataSweepAge = opt.MinMetadataSweepAge
 	lc.Caching.MinIndexSweepAge = opt.MinIndexSweepAge
+	lc.Caching.KeyDerivationAlgorithm = opt.KeyDerivationAlgorithm
+
+	if lc.Caching.KeyDerivationAlgorithm == "" {
+		lc.Caching.KeyDerivationAlgorithm = crypto.DefaultKeyDerivationAlgorithm
```

Comment:
- Add an appropriately named constant for the default algo in the `repo` package.
- Use the constant here and in the CLI flag.